.github/workflows/wix.yml

```diff
@@ -54,13 +52,30 @@ jobs:
     steps:
       - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
 
-      - name: Install dependencies
+      - name: Download IcedTea-Web
         run: |
-          choco install --no-progress wget
+          Invoke-WebRequest -Uri "https://github.com/AdoptOpenJDK/IcedTea-Web/releases/download/${{ matrix.ICEDTEA_WEB_VERSION }}/${{ matrix.ICEDTEA_WEB_VERSION }}.win.bin.zip" `
+            -OutFile "${{ matrix.ICEDTEA_WEB_VERSION }}.win.bin.zip"
+          unzip -q icedtea-web-*.win.bin.zip
+          Remove-Item icedtea-web-*.win.bin.zip
+          Remove-Item icedtea-web-image\share\doc -Recurse
+        working-directory: wix/SourceDir
+        if: ${{ matrix.ICEDTEA_WEB_VERSION }}
 
-      - name: Setup environment variables
-        uses: allenevans/set-env@0f3306034af0ea21dd28983b6e7c1614ee317739 # v3.0.0
-        with:
+      - name: Download Prebuilt JDK/JRE
+        run: |
+          Invoke-WebRequest -Uri "https://github.com/adoptium/temurin${{ matrix.PRODUCT_MAJOR_VERSION }}-binaries/releases/download/${{ matrix.TAG }}/OpenJDK${{ matrix.PRODUCT_MAJOR_VERSION }}U-jdk_${{ matrix.ARCH }}_windows_${{ matrix.JVM }}_${{ matrix.SUB_TAG }}.zip" `
```

Comment:
Can't we use the official Adoptium API for this download rather than reaching into the GitHub store?

Reply:
Argh missed that in my review, @gdams we should fix this.